src/main/java/com/googlecode/jsendnsca/NonBlockingPassiveCheckSenderExceptionHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.googlecode.jsendnsca;
+
+/**
+ * Exception handler to handle exceptions while sending passive checks with the {@link NonBlockingNagiosPassiveCheckSender}.
+ *
+ * @author max.schwaab@gmail.com
+ */
+public interface NonBlockingPassiveCheckSenderExceptionHandler {
```

Comment:
Instead of being a top-level class, wondering if instead it can be a public inner class of NonBlockingPassiveCheckSender called ExceptionHandler.